lua/starfall/instance.lua

```diff
@@ -43,7 +43,10 @@ function SF.Instance.Compile(code, mainfile, player, entity)
 		mainfile = mainfile or "generic"
 		code = { [mainfile] = code }
 	end
-
+	local ok, message = hook.Run("StarfallShouldCompile", code, mainfile, player, entity)
```

Comment:
StarfallCanCompile

Reply:
Might as well also make the return a single line like the other.
`if ok == false then return false, { message = message, traceback = "" } end`